.clang-format

```diff
@@ -0,0 +1,29 @@
+Language: Cpp
+BasedOnStyle: LLVM
+AlignEscapedNewlinesLeft: true
+AlwaysBreakAfterReturnType: TopLevelDefinitions
+BreakBeforeBinaryOperators: All
+BreakBeforeBraces: GNU
+ColumnLimit: 150
+ContinuationIndentWidth: 2
```

Comment:
Is this spaces or tabs? It should be either one tab or 4 spaces (although we only indent using tabs).

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html indicates spaces. I'll change to 4 in the next commit.

Reply:
Changed in 827e967b0